src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -1297,4 +1301,107 @@ uct_rc_mlx5_iface_common_poll_rx(uct_rc_mlx5_iface_common_t *mlx5_common_iface,
     return count;
 }
 
+#if HAVE_IBV_EXP_DM
+/* DM memory should be written by 8 bytes (int64) to eliminate
+ * processor cache issues. To make this used uct_rc_mlx5_dm_copy_data_t
```

Comment:
Is the memory mapped as WB / cacheble ?

Reply:
no ideas. the only limitation there to write data by 8 bytes per iteration